l10n_br_pos/views/pos_template.xml

```diff
@@ -20,13 +20,11 @@
     </xpath>
 </template>
 
-<template id="pos_l10n_br_assets"
-          name="pos_l10n_br_css"
-          inherit_id="point_of_sale.index">
-    <xpath expr="//script[@id='loading-script']" position="before">
-        <link rel="stylesheet" href="/l10n_br_pos/static/src/css/pos.css"/>
-    </xpath>
-</template>
+    <template id="pos_l10n_br_assets" name="pos_l10n_br_css" inherit_id="point_of_sale.index">
```

Comment:
Esta mudança parece desnecessária nesse commit, só mudou whitespace.

Se ela for de fato desnecessária, será possível instalar essa mudança sem atualizar módulos. @lfdivino, confirma isso?

Reply:
Confirmo sim @leorochael, não precisa atualizar o módulo.